src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -195,11 +202,103 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
             }
 
             Response.StatusCode = 401;
-            Response.Headers.Append(HeaderNames.WWWAuthenticate, Options.Challenge);
+
+            var errorDescription = CreateErrorDescription(eventContext.AuthenticateFailure);
```

Comment:
The www-Authenticate header is pretty easy in this scenario, I don't expect people to have trouble creating it.